org.opentosca.planbuilder.postphase.plugin.instancedata/src/org/opentosca/planbuilder/postphase/plugin/instancedata/bpel/Fragments.java

```diff
@@ -113,6 +113,33 @@ public String generateBPEL4RESTLightPUTInstanceState(final String instanceURLVar
 
         return bpel4restString;
     }
+    
+    public String generateBPEL4RESTLightGETInstanceState(final String instanceURLVar,
+                                                          final String ResponseVarName) throws IOException {
+        // BPEL4RESTLightPUT_NodeInstance_State_InstanceDataAPI.xml
+        // <!-- $RequestVarName,$nodeInstanceURLVar -->
```

Comment:
there is no `RequestVarName` variable in that file. The code below is correct, this comment is not